components/suggestion/command_provider/command_provider.tsx

```diff
@@ -100,7 +102,7 @@ export default class CommandProvider extends Provider {
 
         this.store = globalStore;
         this.props = props;
-        this.appCommandParser = new AppCommandParser(this.store as any, intlShim, props.channelId, props.rootId);
+        this.appCommandParser = new AppCommandParser(this.store as any, intlShim, props.channelId, props.teamId, props.rootId);
```

Comment:
This may bite us later, but I feel the code is simpler this way, instead of moving up the parser and many things that require the creation of the parser (like the store or the intl).
At the moment, it should not create many problems, since we are only looking for the base commands, so there are no extra network requests nor states.

Reply:
Also, regarding having more than one AppCommandParser, we have a third one in the command execution action. I think we could consider a bigger refactor on the command parser, to only have one, but I also feel is out of the scope of this PR.